soh/soh/Enhancements/randomizer/3drando/settings.cpp

```diff
@@ -2524,6 +2524,7 @@ namespace Settings {
 
     NumRequiredCuccos.SetSelectedIndex(cvarSettings[RSK_CUCCO_COUNT]);
     BigPoeTargetCount.SetSelectedIndex(cvarSettings[RSK_BIG_POE_COUNT]-1);
+    Shopsanity.SetSelectedIndex(cvarSettings[RSK_SHOPSANITY] == 0 ? 0 : 2);
```

Comment:
this will make it so the 3d rando logic will shuffle items into shops if i'm not mistaken. until we implement the the logic for replacing shop items in soh we should keep this disabled

i'm thinking this means we should just hold off on having the tycoon's wallet until we implement shopsanity

Reply:
Yeah that's what I was thinking. I could probably create a new setting but it'll be pointless once Shopsanity is out. Admittedly though I didn't realize that the items would get shuffled into shops already. It makes sense now that you mention it, items would get shuffled there in seed generation but wouldn't be placed there in game.